talentsearch/app/Http/Controllers/PageContainerController.php

```diff
@@ -2,7 +2,7 @@
 
 namespace App\Http\Controllers;
 
-class DashboardController extends Controller
+class PageContainerController extends Controller
```

Comment:
This should be called TalentSearchController or something like that, not named after a generic container.